conjure_python_client/_http/requests_client.py

```diff
@@ -77,7 +77,7 @@ def _request(self, *args, **kwargs):
         try:
             _response.raise_for_status()
         except HTTPError as e:
-            if e.response is not None and e.response.content is not None:
+            if e.response is not None:
```

Comment:
why was this changed?

Reply:
We want to handle a response with no content with the ConjureHTTPError so we can show the HTTP message & trace ID. Without that it will not reach the block I modified to add the trace ID.